content/en/agent/guide/agent-configuration-files.md

```diff
@@ -73,14 +73,16 @@ Prior releases of Datadog Agent stored configuration files in `/dd-agent/conf.d/
 
 ### Checks configuration files for Agent 6
 
-An example for each Agent check configuration files can be found in the `conf.yaml.example` file in the corresponding `<CHECK_NAME>.d/` folder. Rename this file to `conf.yaml` in order to enable the associated check. Note that the Agent loads any valid YAML file contained in the folder: `/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`. This way, complex configurations can be broken down into multiple files. For example, a configuration for the `http_check` might look like this:
+An example for each Agent check configuration file is found in the `conf.yaml.example` file in the corresponding `<CHECK_NAME>.d/` folder. Rename this file to `conf.yaml` to enable the associated check. Note that the Agent loads valid YAML files contained in the folder: `/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`. This allows complex configurations to be broken down into multiple files. For example, a configuration for the `http_check` might look like this:
 
 ```text
 /etc/datadog-agent/conf.d/http_check.d/
 ├── backend.yaml
 └── frontend.yaml
 ```
 
+A special case are YAML files with the suffix `.default`. These files are loaded by the Agent by default and help define the core set of checks that are always enabled. To turn off one of these checks, change the suffix to `.disabled` or remove the file altogether.  To customize these checks, copy the `conf.yaml.default` file to `conf.yaml` and make any desired changes, and the `.default` file will be ignored.
+
```

Comment:
Users should not use `.default` as their base configuration as the `.example` contains all the doc strings. `.default` should just be ignored by users.

What do you think of:

```suggestion
A special case are YAML files with the suffix `.default`. These files are loaded by the Agent by default and help define the core set of checks that are always enabled (CPU, memory, uptime ...). They are ignored if any other configuration are found for that check, therefor you can safely ignore them. If you want to disable one of the default check simply remove that file. To configure these checks, `conf.yaml.example` should be use as a base.
```